### DISCUSSION TITLE
How to delete my account on https://cssbattle.dev ?

### DISCUSSION THREAD

**Discussion by BinaryHammer** [cssbattle/forum#286] [Bugs reports]
There is no way to delete my https://cssbattle.dev account.

I have no Twitter account. Creating and deleting a Twitter account just for deleting my CSS Battle account is even more of a hassle.
How do I initiate the account deletion?

Please delete my account associated with my email, but where do I tell the admins of https://cssbattle.dev my email?

Since the Email is personal information, it should be possible to initiate a delete process.

**Comment by chinchang** [comment]
Hey @BinaryHammer 
Sure! Please send a deletion request to contact@cssbattle.dev
Sorry about not having an automated way of doing this right now from the UI.
